### DISCUSSION TITLE
Please spell `macOS` as such in the Runners view

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#28643] [Actions] [answered]
<img width="895" alt="image" src="https://user-images.githubusercontent.com/2119212/183278420-c27e7adb-b16e-4da2-8cbe-faf5d2c4c438.png">

<details><summary>Text</summary>

> Runners / GitHub-hosted runners
> All jobs usage
20/20
Linux 16
Windows 2
**MacOS** 2
To increase your concurrency limit, upgrade your [GitHub plan.](https://docs.github.com/billing/managing-billing-for-github-actions)

> Labels
windows-2016
windows-2019
windows-2022
windows-latest
ubuntu-18.04
ubuntu-20.04
ubuntu-latest
macos-10.15
macos-11
macos-latest

</details>

The brand is `macOS`. For an example of how the vendor uses it, see: https://www.apple.com/ca/macos/monterey/

**Comment by dylanatsmith** [comment, marked as answer]
Thanks for flagging this @jsoref. I’ve opened a PR and you should see this updated today.
